process/block/export_test.go

```diff
@@ -551,6 +551,11 @@ func (bp *baseProcessor) CheckConstructionStateAndIndexesCorrectness(mbh data.Mi
 	return checkConstructionStateAndIndexesCorrectness(mbh)
 }
 
+// SaveEpochStartInfoToStaticStorage -
+func (bp *baseProcessor) SaveEpochStartInfoToStaticStorage(header data.HeaderHandler, marshalledHeader []byte, body *block.Body) {
```

Comment:
can you move this as well on the meta processor?

Reply:
done